.github/workflows/notify-studio-pr.yml

```diff
@@ -20,17 +10,21 @@ jobs:
       - name: Init
         id: init
         run: |
-          if [ ${{ github.event.action }} == 'github-studio-pr' ]
-          then
               echo "##[set-output name=tagVersion;]$(echo ${{ github.event.client_payload.releaseVersion }})"
               echo "##[set-output name=studioBaseBranch;]$(echo ${{ github.event.client_payload.baseBranch }})"
-          else
-              echo "##[set-output name=tagVersion;]$(echo ${{ github.event.inputs.version }})"
-              echo "##[set-output name=studioBaseBranch;]$(echo ${{ github.event.inputs.baseBranche }})"
-              echo "github.ref => ${{ github.ref }})"
-          fi
-      - name: Print parameter
-        id: initParameter
-        run: |
-          echo "studioBaseBranch => ${{ steps.init.outputs.studioBaseBranch}}"
-          echo "tagVersion => ${{ steps.init.outputs.tagVersion}}"
+      - name: Send message to Slack channel
+        uses: 8398a7/action-slack@v3
+      with:
+        status: custom
+        fields: workflow,job,commit,repo,ref
+        custom_payload: |
+          {
+            username: 'action-slack',
+            icon_emoji: ':octocat:',
+            attachments: [{
+              color: 'good',
+              text: `UI-Designer release ${{ github.event.client_payload.releaseVersion }} success. Next step: Open PR in bonita-studio on branch ${{github.event.client_payload.baseBranch}} repository and close nexus release`,
```

Comment:
If you could remain the file to update in Studio, and provide the link for Nexus, il would be even better!

Reply:
I need to update it, we don't need to go on nexus now ;)